### DISCUSSION TITLE
Scripting Python Foundations Homework

### DISCUSSION THREAD

**Discussion by ZoeLeBlanc** [CultureAsData-UIUC/is310-fall-2024#7] [Assignments]
Please add a link below to your Scripting Python Foundations Homework [https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/creating-curating-humanities-data/02-python-refresher-advanced#homework-scripting-python-foundations-optional-but-recommended](https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/creating-curating-humanities-data/02-python-refresher-advanced#homework-scripting-python-foundations-optional-but-recommended). Remember this assignment is optional. If you decide to complete it, it will be added to your grade as extra credit but if you do not complete it, it will not be counted against you.

**Comment by kohtas2** [comment]
https://github.com/kohtas2/is310-coding-assignments/blob/main/in-class-coding/first_script.py

**Comment by GloriaLiu930** [comment]
https://github.com/GloriaLiu930/is310-coding-assignments/blob/main/scripting-python-foundations/first_script.py

**Comment by esr27** [comment]
https://github.com/esr27/is310-coding-assignments/blob/main/first_script.py

**Comment by yosefstifanos** [comment]
https://github.com/yosefstifanos/is310-coding-assignments/tree/main/first_script.py%20homework%20(extra%20credit)

**Comment by jasonmorris8** [comment]
https://github.com/jasonmorris8/is310-coding-assignments/blob/main/first_script.py

**Comment by ahmadsawwan** [comment]
https://github.com/ahmadsawwan/ahmadsawwan-is310-coding-assignments/blob/main/first_script.py

**Comment by yunchaecho2** [comment]
https://github.com/yunchaecho2/is310-coding-assignments/blob/main/python-script/first_script.py

**Comment by LucasWashor** [comment]
https://github.com/LucasWashor/is310-coding-assignments/blob/main/python-scripting/first_script.py

**Comment by henRotunno** [comment]
https://github.com/henRotunno/henRotunno.github.io/blob/main/moviescrape.py

**Reply by ZoeLeBlanc** [reply]
Hi @henRotunno ! This looks good but I'm a bit confused why this assignment is in your `henRotunno.github.io` repo and not your `is310-coding-assignments` repo. I'll count this as complete but please have all your assignments in the `is310-coding-assignments` (with the exception for the link to your website for the Doing it Live Homework). Thanks and good work 👍

**Comment by shoirathi** [comment]
https://github.com/shoirathi/is310-coding-assignments/blob/main/python-scripting/first_script.py

**Comment by tinayang922** [comment]
https://github.com/tinayang922/is310-coding-assignments/blob/main/optional-hw/first_script.py

**Comment by elee33** [comment]
https://github.com/elee33/is310-coding-assignments/blob/main/python3%20first_script.py

**Comment by theajk2** [comment]
https://github.com/theajk2/is310-coding-assignments/blob/main/first_script.py

**Reply by ZoeLeBlanc** [reply]
Hi @theajk2 ! Just fyi your script doesn't currently work. When I try to run it, I get the following error:

![Screenshot 2024-10-09 at 4 01 02 PM](https://github.com/user-attachments/assets/03c122c1-dc22-4f63-ada6-5d52a217f2e9)

Hopefully you can see the issue and feel free to ping me here or on Discord if you are still having issues (essentially you just need to make a small fix for the script to work)

**Reply by theajk2** [reply]
I think I have fixed the issue with the function by changing the argument it takes.

**Comment by divyaj5** [comment]
https://github.com/divyaj5/is310-coding-assignments/blob/master/scripting-python-foundations/first_script.py

**Comment by yukthas2** [comment]
https://github.com/yukthas2/is310-coding-assignments/tree/main/python-scripting
